.env-example

```diff
@@ -0,0 +1,6 @@
+export PSQL_USER=hivemind_chitter
```

Comment:
God knows what all this does but keep on slaying the game Paul

Reply:
😆